content/en/continuous_integration/pipelines/custom.md

```diff
@@ -90,7 +91,17 @@ To send pipeline events programmatically to Datadog, ensure that your [`DD_API_K
    EOF
    {{< /code-block >}}
 
-1. After sending your pipeline event to Datadog, you can integrate additional event types such as `stage`, `job`, and `step`. For more information, see the [Send Pipeline Event endpoint][1].
+3. After sending your pipeline event to Datadog, you can integrate additional event types such as `stage`, `job`, and `step`. For more information, see the [Send Pipeline Event endpoint][1].
+
+## Running pipelines
+Pipeline events sent with the `status` set to `running` have the same `unique_id` as the final pipeline event.
+Additionally, running pipelines can be updated by adding more information while they are still running. Essentially, a pipeline consists of the following events:
```

Comment:
```suggestion
Pipeline events sent with the `status` set to `running` have the same `unique_id` as the final pipeline event. Running pipelines can be updated by adding more information while they are still running. A running pipeline consists of the following events:
```

Reply:
It sounds like this section is not about updating running pipelines, but rather describing them. Is this correct?

Reply:
Correct, it is about describing them. They have the API documentation for the details